coc/client.py

```diff
@@ -257,9 +256,15 @@ async def login(self, email: str, password: str) -> None:
         """
         self.http = http = self._create_client(email, password)
         await http.create_session(self.connector, self.timeout)
-        await http.initialise_keys()
-        self._create_holders()
 
+        try:
+            await http.initialise_keys()
+        except Exception as error:
+            LOG.error("Invalid credentials\n%s", error)
+            await http.close()
+            raise
```

Comment:
Does this re-raise the InvalidCredentials error? For people who don't have logging setup a detailed exception is important because it's ambiguous why it failed otherwise

Reply:
Yeah it re-raises coc.InvalidCredentials so the user should already be tracking. You can see it in action in the examples